thirdeye/thirdeye-frontend/app/pods/partials/rootcause/dimensions/template.hbs

```diff
@@ -6,12 +6,15 @@
     {{#subtablist.tab name="heatmap"}}Heatmap{{/subtablist.tab}}
   {{/subtab.tablist}}
 
-  {{#if model.isDevEnv}} {{!--  Remove conditional when table view is ready  --}}
+  {{#if model.isDevEnv}} {{!--  Remove conditional when table view is ready --}}
```

Comment:
NIP: {{!-- TODO:  Remove conditional when table view is ready --}}

Reply:
Just need to confirm data is correct before releasing it.